src/components/ModalPortal.js

```diff
@@ -36,6 +41,13 @@ export default {
   },
 
   methods: {
+    getModals() {
+      const stack = this.$modal.stack.slice().reverse()
+      const nonOverlayIndex = stack.findIndex(({ overlay }) => !overlay)
```

Comment:
Can we use `for` statement to find the index instead because `findIndex` is not on IE? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex